### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by vchrizz**
copied from OONF and modified to suit olsrd.
Signed-Off-By: Christoph Loesch <mail@chil.at>

### CODE REVIEW

**Comment by fhuberts** [comment]
I need a Signed-Off-By before merging

**Comment by fhuberts** [comment]
in the commit please

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Download and install the neccessary build requirements
+
+For Debian you will mostly need the following ones:
+  * build-essentials: ``sudo apt-get install build-essential dpkg-dev``
+
+## Compiling
+
+  * ``cd olsrd``
+  * ``make``
+  * (plugins) ``cd lib; for i in $(ls); do cd "$i"; make; cd ..; done``
```
Comment (fhuberts) [inline README.md]: make libs

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Configuring OLSRd
+
+## Starting OLSRd
+
+Assuming your interfaces you want olsrd to listen on are ``eth0, wlan0 and lo`` you could start it like this:
+
+  * ``sudo ./olsrd_static -i eth0 wlan0 lo -d 0 -f olsrd.conf``
```
Comment (fhuberts) [inline README.md]: did you check this?
I think it should be "... -i eth0 -i wlan0 -i lo ...."

Comment (fhuberts) [reply README.md]: The -f is not needed; it will look in /etc/olsrd/olsrd.conf

File: README.md (hunk 1 above)
Comment (vchrizz) [reply README.md]: updated.

File: README.md (hunk 2 above)
Comment (vchrizz) [reply README.md]: see e.g. https://www.apt-browse.org/browse/debian/wheezy/main/i386/olsrd/0.6.2-2.1/file/etc/olsrd/olsrd.conf
using the debian package the interfaces are defined this way and this worked for me some time ago.
i didnt try multiple -i options though.

Comment (vchrizz) [reply README.md]: removed -f option.

**Comment by vchrizz** [comment]
using `make libs` i just noticed a plugin that didnt compile because of missing file gps.h
on debian i had to install package libgps-dev for this to work.
not sure if this is worth to mention in the readme?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+(see https://github.com/OLSR/olsrd)
+
+ * ``git clone https://github.com/OLSR/olsrd.git``
+
+### Pre-requisites
+
+Download and install the neccessary build requirements
+
+For Debian you will mostly need the following ones:
+  * build-essentials: ``sudo apt-get install build-essential dpkg-dev``
```
Comment (fhuberts) [inline README.md]: Add: flex bison  libgps-dev

**Comment by PolynomialDivision** [comment]
This PR seems abandoned. I will close. Feel free to reopen.

**Pull request closed without merging**